Obsidian/Client.cs

```diff
@@ -195,6 +195,12 @@ public async Task StartConnectionAsync()
                             var loginStart = LoginStart.Deserialize(data);
 
                             string username = config.MulitplayerDebugMode ? $"Player{Globals.Random.Next(1, 999)}" : loginStart.Username;
+                            
+                            if (config.WhitelistEnabled && !config.WhitelistedNicknames.Contains(username))
```

Comment:
This only checks for username. PR is fine otherwise, but would you mind adding an additional check for UUID? UUIDs are generated _after_ the online mode check.

In offline mode you'd check for nickname, yeah, but when in online mode you'd check for UUID. Using the same array is fine.

Line 205 fetches a user from Mojang's API. This response (`var user`) should contain a field named `id`.

It also still needs to convert usernames to uuids before launching the server, but it's fine if you choose to not do that for this PR yet.

Reply:
> This only checks for username. PR is fine otherwise, but would you mind adding an additional check for UUID? UUIDs are generated _after_ the online mode check.
> 
> In offline mode you'd check for nickname, yeah, but when in online mode you'd check for UUID. Using the same array is fine.
> 
> Line 205 fetches a user from Mojang's API. This response (`var user`) should contain a field named `id`.
> 
> It also still needs to convert usernames to uuids before launching the server, but it's fine if you choose to not do that for this PR yet.

Fixed I guess, I can't check if online mode whitelist works 'cause my MC account was stolen